sdk/azidentity/chained_token_credential.go

```diff
@@ -51,7 +50,7 @@ func (c *ChainedTokenCredential) GetToken(ctx context.Context, opts azcore.Token
 	if token == nil && len(errList) == 0 {
 		return nil, nil
 	}
-	return nil, &ChainedCredentialError{ErrorList: errList}
```

Comment:
Is `ChainedCredentialError` still used someplace or can its type definition be removed now?

Reply:
It's still used when we fail to find any available credentials in GetToken for ChainedTokenCredentials.

Reply:
Actually we wont be using ChainedCredentialError anymore. I will add a task to remove it from the code in my next PR.

Reply:
Here's the issue: #6549